app/util/NotifyingAtomPublisher.scala

```diff
@@ -0,0 +1,48 @@
+package util
+
+import com.amazonaws.services.sns.AmazonSNS
+import com.amazonaws.services.sns.model.PublishRequest
+import com.gu.atom.publish.AtomPublisher
+import com.gu.contentatom.thrift.{ContentAtomEvent, EventType}
+import org.joda.time.{DateTime, DateTimeZone}
+import play.api.libs.json.{Json, Writes}
+
+import scala.util.Try
+
+class NotifyingAtomPublisher(isLive: Boolean, topicArn: String, underlying: AtomPublisher, sns: AmazonSNS) extends AtomPublisher {
+  override def publishAtomEvent(event: ContentAtomEvent): Try[Unit] = {
+    underlying.publishAtomEvent(event).map { _ =>
+      val notification = SimpleContentUpdate.fromEvent(event, isLive)
+      val json = Json.stringify(Json.toJson(notification))
+
+      // TODO MRB: should we have a new message-type called atom update?
```

Comment:
It can't hurt 😅 though at this moment it's redundant with the topic. I suppose the question is if instead of two topic+queue there could be only one?

Reply:
For the price of an extra SQS I think it's best to keep them separated for now whilst we experiment. Doesn't look like we filter on subject but good to know the feature is there

Reply:
I think I will change the serialisation reader in Pubflow to allow us to use `id` as well as `composerId` in `SimpleContentUpdate` just to avoid it getting confusing

Reply:
yes, makes sense.